frontend/src/components/widgets/LinkToUserBoard.js

```diff
@@ -1,18 +1,20 @@
 import React from "react";
 import { routes } from "../../routes";
-
 import BaseButtonLink from "./BaseButtonLink";
+import { useStyles } from "./BaseButtonLink";
```

Comment:
Please rather style things in place. Spagetti styling makes things hard to reason about. 

Importing and exporting styles from all over the place makes it hard to know what things do.

If styles are used in 1 component put them in that component.

Global styles should not be defined in strange littel components.